auth/src/main/java/feast/auth/authorization/Keto/KetoAuthorizationProvider.java

```diff
@@ -57,20 +59,28 @@ public KetoAuthorizationProvider(Map<String, String> options) {
    * @return AuthorizationResult result of authorization query
    */
   public AuthorizationResult checkAccess(String project, Authentication authentication) {
-    String email = getEmailFromAuth(authentication);
+    String subject = (String) this.options.get("subject");
+    if ((subject == null) || (subject.isEmpty())) subject = "email";
+    String subjectValue = getSubjectFromAuth(authentication, subject);
```

Comment:
no need to pass in the claim name since `this.options` can be referenced in the method